src/test/java/com/cloudbees/jenkins/support/impl/SlaveCommandStatisticsTest.java

```diff
@@ -36,15 +36,16 @@
 import java.io.IOException;
 import java.util.logging.Level;
 import jenkins.MasterToSlaveFileCallable;
-import static org.hamcrest.Matchers.*;
 import org.junit.Test;
-import static org.junit.Assert.*;
 import org.junit.Rule;
 import org.jvnet.hudson.test.Issue;
 import org.jvnet.hudson.test.JenkinsRule;
 import org.jvnet.hudson.test.LoggerRule;
 import org.jvnet.hudson.test.TestBuilder;
 
+import static org.hamcrest.Matchers.*;
```

Comment:
Why this one if the actual doesn't change?

Reply:
Because now in all test classes the block of static imports is placed below non-static ones.

Reply:
But these are not movements but additions? I mean, if they were not necessary before, why are they now?

Reply:
`import static org.hamcrest.Matchers.*;` is moved from line 39 to line 46.

Reply:
`import static org.junit.Assert.*;` is moved from line 41 to line 46 and the not required star import was simplified to importing only the single required method.

Reply:
Sorry, I didn't see the other ones :(